v03_pipeline/lib/model/cached_reference_dataset_query.py

```diff
@@ -13,7 +13,7 @@
 )
 
 
-class CachedReferenceDatasetQuery(Enum):
+class CachedReferenceDatasetQuery(str, Enum):
```

Comment:
`StrEnum` is available in python3.11 only.  It's necessary for the json serialization in the endpoint.